### ISSUE TITLE
Visually highlight focused elements (for tabbing)

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/opencast-studio#629]
The focused element should have a clear indication that it is focused. Otherwise using the page without a mouse is almost impossible.

**Comment by naweidner** [comment]
Fixed with #931 and #933
